### PR TITLE
feature(tiger): Добавит модальник

### PR DESCRIPTION
**Pull request by dontuse**

### CODE REVIEW

File: src/Dialog/Dialog.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  render() {
+    return (
+      <RcDialog
+        animation='slide-fade'
+        maskAnimation='fade'
+        {...this.props}
+      >
+        {this.props.children}
+      </RcDialog>
```
Comment (apress-hound) [inline src/Dialog/Dialog.js]: Unclosed regular expression.<br>Unrecoverable syntax error. (68% scanned).

File: src/Button/e-button.scss [hunk 2]
```diff
@@ -1,8 +1,11 @@
 .e-button {
-  box-sizing: border-box; line-height: 1.42857143; border-radius: 3px; background-color: #028dd8; display: inline-block; color:  #fff; cursor: pointer; padding: 6px 20px; white-space: nowrap; border: 1px solid transparent; font-style: inherit; font-size: 12px; text-align: center;
-  &:hover { background-color: #00a6ff; }
+  box-sizing: border-box; line-height: 1.42857143; border-radius: 3px; background-color: #028dd8; display: inline-block; color:  #fff; cursor: pointer; padding: 6px 20px; white-space: nowrap; border: 1px solid transparent; font-style: inherit; font-size: 12px; text-align: center; transition: opacity 0.3s;
+  &:hover { opacity: 0.85 }
```
Comment (apress-hound) [inline src/Button/e-button.scss]: Declaration should be terminated by a semicolon<br>`0.85` should be written without a leading zero as `.85`

File: src/Button/e-button.scss [hunk 3]
```diff
@@ -1,8 +1,11 @@
 .e-button {
-  box-sizing: border-box; line-height: 1.42857143; border-radius: 3px; background-color: #028dd8; display: inline-block; color:  #fff; cursor: pointer; padding: 6px 20px; white-space: nowrap; border: 1px solid transparent; font-style: inherit; font-size: 12px; text-align: center;
-  &:hover { background-color: #00a6ff; }
+  box-sizing: border-box; line-height: 1.42857143; border-radius: 3px; background-color: #028dd8; display: inline-block; color:  #fff; cursor: pointer; padding: 6px 20px; white-space: nowrap; border: 1px solid transparent; font-style: inherit; font-size: 12px; text-align: center; transition: opacity 0.3s;
```
Comment (apress-hound) [inline src/Button/e-button.scss]: Colon after property should be followed by one space<br>`0.3` should be written without a leading zero as `.3`

File: src/Dialog/rc-dialog.scss [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+}
+@keyframes rcDialogFadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+@-webkit-keyframes rcDialogFadeOut {
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Avoid vendor prefixes.

File: src/Dialog/rc-dialog.scss [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+.rc-dialog-fade-enter,
+.rc-dialog-fade-appear { opacity: 0; animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
+.rc-dialog-fade-leave { animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
+.rc-dialog-fade-enter.rc-dialog-fade-enter-active,
+.rc-dialog-fade-appear.rc-dialog-fade-appear-active { animation-name: rcDialogFadeIn; animation-play-state: running; }
+.rc-dialog-fade-leave.rc-dialog-fade-leave-active { animation-name: rcDialogFadeOut; animation-play-state: running; }
+
+@-webkit-keyframes rcDialogFadeIn {
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Avoid vendor prefixes.

File: src/Dialog/rc-dialog.scss [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    transform: translate(0, -25%);
+  }
+}
+
+.rc-dialog-fade-enter,
+.rc-dialog-fade-appear { opacity: 0; animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
+.rc-dialog-fade-leave { animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
+.rc-dialog-fade-enter.rc-dialog-fade-enter-active,
+.rc-dialog-fade-appear.rc-dialog-fade-appear-active { animation-name: rcDialogFadeIn; animation-play-state: running; }
+.rc-dialog-fade-leave.rc-dialog-fade-leave-active { animation-name: rcDialogFadeOut; animation-play-state: running; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Merge rule `.rc-dialog-fade-leave.rc-dialog-fade-leave-active` with rule on line 77

File: src/Dialog/rc-dialog.scss [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    transform: translate(0, 0);
+  }
+  100% {
+    transform: translate(0, -25%);
+  }
+}
+
+.rc-dialog-fade-enter,
+.rc-dialog-fade-appear { opacity: 0; animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
+.rc-dialog-fade-leave { animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0.3` should be written without a leading zero as `.3`<br>`0.55` should be written without a leading zero as `.55`<br>`0.2` should be written without a leading zero as `.2`

File: src/Dialog/rc-dialog.scss [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+  0% {
+    transform: translate(0, 0);
+  }
+  100% {
+    transform: translate(0, -25%);
+  }
+}
+
+.rc-dialog-fade-enter,
+.rc-dialog-fade-appear { opacity: 0; animation-duration: 0.3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2); animation-play-state: paused; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0.3` should be written without a leading zero as `.3`<br>`0.55` should be written without a leading zero as `.55`<br>`0.2` should be written without a leading zero as `.2`

File: src/Dialog/rc-dialog.scss [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+}
+@keyframes rcDialogSlideFadeIn {
+  0% {
+    transform: translate(0, -25%);
+  }
+  100% {
+    transform: translate(0, 0);
+  }
+}
+@-webkit-keyframes rcDialogSlideFadeOut {
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Avoid vendor prefixes.

File: src/Dialog/rc-dialog.scss [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+.rc-dialog-slide-fade-appear,
+.rc-dialog-slide-fade-leave { animation-duration: .3s; animation-fill-mode: both; animation-timing-function: ease-out; animation-play-state: paused; }
+
+.rc-dialog-slide-fade-enter.rc-dialog-slide-fade-enter-active,
+.rc-dialog-slide-fade-appear.rc-dialog-slide-fade-appear-active { animation-name: rcDialogSlideFadeIn; animation-play-state: running; }
+
+.rc-dialog-slide-fade-leave.rc-dialog-slide-fade-leave-active { animation-name: rcDialogSlideFadeOut; animation-play-state: running; }
+
+
+@-webkit-keyframes rcDialogSlideFadeIn {
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Avoid vendor prefixes.

File: src/Dialog/rc-dialog.scss [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+  &-body { padding: 0px 60px; }
+  &-footer { padding: 40px 60px; }
+  &-button { margin-right: 15px; }
+  &-mask {
+    position: fixed; top: 0; right: 0; left: 0; bottom: 0; background-color: #373737; background-color: rgba(55, 55, 55, 0.6); height: 100%; z-index: 1050;
+    &-hidden { display: none; }
+  }
+  @media (min-width: 768px) {
+    & { width: 600px; margin: 30px auto; }
+    &-content { box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5); }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0.5` should be written without a leading zero as `.5`

File: src/Dialog/rc-dialog.scss [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
+  &-footer { padding: 40px 60px; }
+  &-button { margin-right: 15px; }
+  &-mask {
+    position: fixed; top: 0; right: 0; left: 0; bottom: 0; background-color: #373737; background-color: rgba(55, 55, 55, 0.6); height: 100%; z-index: 1050;
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Property `background-color` already defined on line 20<br>`0.6` should be written without a leading zero as `.6`

File: src/Dialog/rc-dialog.scss [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, 0.52); background-clip: padding-box; outline: 0; }
+  &-header { padding: 60px; padding-top: 40px; padding-bottom: 25px; }
+  &-close {
+    cursor: pointer; position: absolute; right: 20px; top: 20px; line-height: 1; text-decoration: none; text-indent: -9999999px; overflow: hidden; width: 24px; height: 24px; border: 2px solid #9c9da1; border-radius: 12px; display: flex; align-items: center; justify-content: center; background: #fff; padding: 0;
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0px` should be written without units as `0`

File: src/Dialog/rc-dialog.scss [hunk 14]
```diff
@@ -0,0 +1,113 @@
+@import '../styles/mixins/icons';
+
+.rc-dialog {
+  box-sizing: border-box; position: relative; width: auto; margin: 10px; font-family: OpenSans;
+  & * { box-sizing: border-box; }
+  &-wrap { position: fixed; overflow: auto; top: 0; right: 0; bottom: 0; left: 0; z-index: 1050; -webkit-overflow-scrolling: touch; outline: 0; }
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Property `background-color` already defined on line 7<br>Color `#ffffff` should be written as `#fff`

File: src/Dialog/example.js [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+        wrapClassName = 'center';
+      }
+      dialog = (
+        <Dialog
+          visible={this.state.visible}
+          wrapClassName={wrapClassName}
+          onClose={this.onClose}
+          style={style}
+          mousePosition={this.state.mousePosition}
+          title={<div>Загрузка фотографий</div>}
```
Comment (apress-hound) [inline src/Dialog/example.js]: Unclosed regular expression.<br>Unrecoverable syntax error. (61% scanned).

File: src/Dialog/example.js [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+    });
+  }
+
+  center = (e) => {
+    this.setState({
+      center: e.target.checked,
+    });
+  }
+
+  render() {
```
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+    this.setState({
+      width: this.state.width === 600 ? 800 : 600,
+    });
+  }
+
+  center = (e) => {
+    this.setState({
+      center: e.target.checked,
+    });
+  }
```
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/rc-dialog.scss [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+    transform: translate(0, -25%);
+  }
+}
+
+.rc-dialog-fade-enter,
+.rc-dialog-fade-appear { opacity: 0; animation-duration: .3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(.55, 0, .55, .2); animation-play-state: paused; }
+.rc-dialog-fade-leave { animation-duration: .3s; animation-fill-mode: both; animation-timing-function: cubic-bezier(.55, 0, .55, .2); animation-play-state: paused; }
+.rc-dialog-fade-enter.rc-dialog-fade-enter-active,
+.rc-dialog-fade-appear.rc-dialog-fade-appear-active { animation-name: rcDialogFadeIn; animation-play-state: running; }
+.rc-dialog-fade-leave.rc-dialog-fade-leave-active { animation-name: rcDialogFadeOut; animation-play-state: running; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Merge rule `.rc-dialog-fade-leave.rc-dialog-fade-leave-active` with rule on line 53

File: src/Dialog/example.js [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+    this.setState({
+      destroyOnClose: e.target.checked,
+    });
+  }
+
+  changeWidth = () => {
+    this.setState({
+      width: this.state.width === 600 ? 800 : 600,
+    });
+  }
```
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+    this.setState({
+      visible: false,
+    });
+  }
+
+  onDestroyOnCloseChange = (e) => {
+    this.setState({
+      destroyOnClose: e.target.checked,
+    });
+  }
```
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js [hunk 21]
```diff
@@ -0,0 +1,106 @@
...
+      },
+      visible: true,
+    });
+  }
+
+  onClose = () => {
+    // console.log(e);
+    this.setState({
+      visible: false,
+    });
```
Comment (apress-hound) [inline src/Dialog/example.js]: Unexpected '}'.<br>Expected an identifier and instead saw '}'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Expected an identifier and instead saw ';'.

File: src/Dialog/example.js [hunk 22]
```diff
@@ -0,0 +1,106 @@
...
+export default class DialogExample extends React.Component {
+
+  state = {
+    visible: false,
+    width: 600,
+    destroyOnClose: false,
+    center: false,
+  }
+
+  onClick = (e) => {
```
Comment (apress-hound) [inline src/Dialog/example.js]: 'onClick' is defined but never used.

File: src/Dialog/rc-dialog.scss [hunk 23]
```diff
@@ -0,0 +1,84 @@
...
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Merge rule `.rc-dialog-fade-leave.rc-dialog-fade-leave-active` with rule on line 62

File: src/Dialog/rc-dialog.scss [hunk 24]
```diff
@@ -0,0 +1,84 @@
...
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
+  &-header { padding: 60px; padding-top: 40px; padding-bottom: 25px; }
+  &-close {
+    cursor: pointer; position: absolute; right: 20px; top: 20px; line-height: 1; text-decoration: none; text-indent: -9999999px; overflow: hidden; width: 24px; height: 24px; border: 2px solid #9c9da1; border-radius: 12px; display: flex; align-items: center; justify-content: center; background: #fff; padding: 0;
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0px` should be written without units as `0`

File: src/Dialog/rc-dialog.scss [hunk 25]
```diff
@@ -0,0 +1,84 @@
+@import '../styles/mixins/icons';
+
+.rc-dialog {
+  box-sizing: border-box; position: relative; width: auto; margin: 10px; font-family: OpenSans;
+  & * { box-sizing: border-box; }
+  &-wrap { position: fixed; overflow: auto; top: 0; right: 0; bottom: 0; left: 0; z-index: 1050; -webkit-overflow-scrolling: touch; outline: 0; }
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Property `background-color` already defined on line 7<br>Colors must have either three or six digits: `#ffff`

File: src/Dialog/example.js (hunk 15 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Unclosed regular expression.<br>Unrecoverable syntax error. (61% scanned).

File: src/Dialog/example.js (hunk 16 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 17 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 19 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 20 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 21 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Unexpected '}'.<br>Expected an identifier and instead saw '}'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Expected an identifier and instead saw ';'.

File: src/Dialog/example.js (hunk 22 above)
Comment (apress-hound) [inline src/Dialog/example.js]: 'onClick' is defined but never used.

File: src/Dialog/example.js [hunk 26]
```diff
@@ -0,0 +1,106 @@
...
+    this.setState({
+      mousePosition: {
+        x: e.pageX,
+        y: e.pageY,
+      },
+      visible: true,
+    });
+  }
+
+  onClose = () => {
```
Comment (apress-hound) [inline src/Dialog/example.js]: Expected ')' to match '}' from line 12 and instead saw 'onClose'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Expected an identifier and instead saw '='.<br>Expected an operator and instead saw '('.<br>Expected '(' and instead saw ')'.<br>Expected an identifier and instead saw '=>'.

File: src/Dialog/example.js [hunk 27]
```diff
@@ -0,0 +1,106 @@
...
+import Button from '../Button/Button';
+
+export default class DialogExample extends React.Component {
+
+  state = {
+    visible: false,
+    width: 600,
+    destroyOnClose: false,
+    center: false,
+  }
```
Comment (apress-hound) [inline src/Dialog/example.js]: Expected '(' and instead saw '}'.

File: src/Dialog/example.js [hunk 28]
```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Dialog from './Dialog';
+import Button from '../Button/Button';
+
+export default class DialogExample extends React.Component {
+
+  state = {
```
Comment (apress-hound) [inline src/Dialog/example.js]: Class properties must be methods. Expected '(' but instead saw '='.

File: src/Dialog/rc-dialog.scss [hunk 29]
```diff
@@ -0,0 +1,63 @@
...
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
+  &-footer { padding: 40px 60px; }
+  &-button { margin-right: 15px; }
+  &-mask {
+    position: fixed; top: 0; right: 0; left: 0; bottom: 0; background-color: #373737; background-color: rgba(55, 55, 55, .6); height: 100%; z-index: 1050;
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Property `background-color` already defined on line 21

File: src/Dialog/rc-dialog.scss [hunk 30]
```diff
@@ -0,0 +1,63 @@
...
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
+  &-header { padding: 60px; padding-top: 40px; padding-bottom: 25px; }
+  &-close {
+    cursor: pointer; position: absolute; right: 20px; top: 20px; line-height: 1; text-decoration: none; text-indent: -9999999px; overflow: hidden; width: 24px; height: 24px; border: 2px solid #9c9da1; border-radius: 12px; display: flex; align-items: center; justify-content: center; background: #fff; padding: 0;
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0px` should be written without units as `0`

File: src/Dialog/rc-dialog.scss [hunk 31]
```diff
@@ -0,0 +1,63 @@
+@import '../styles/mixins/icons';
+
+.rc-dialog {
+  $root: &;
+  box-sizing: border-box; position: relative; width: auto; margin: 10px; font-family: OpenSans;
+  & * { box-sizing: border-box; }
+  &-wrap { position: fixed; overflow: auto; top: 0; right: 0; bottom: 0; left: 0; z-index: 1050; -webkit-overflow-scrolling: touch; outline: 0; }
+  &-content { background-color: #fff; border-radius: 5px; background-color: #ffff; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: Property `background-color` already defined on line 8<br>Colors must have either three or six digits: `#ffff`

**Comment by dontuse** [comment]
@Kupov

File: src/Dialog/rc-dialog.scss [hunk 32]
```diff
@@ -0,0 +1,63 @@
...
+  &-content { background-color: #fff; border-radius: 5px; box-shadow: 0 0 9px 0 rgba(59, 61, 77, .52); background-clip: padding-box; outline: 0; }
+  &-header { padding: 60px; padding-top: 40px; padding-bottom: 25px; }
+  &-close {
+    cursor: pointer; position: absolute; right: 20px; top: 20px; line-height: 1; text-decoration: none; text-indent: -9999999px; overflow: hidden; width: 24px; height: 24px; border: 2px solid #9c9da1; border-radius: 12px; display: flex; align-items: center; justify-content: center; background: #fff; padding: 0;
+    &-x:after { @include close-icon(#9c9da1); content: ''; display: block; width: 12px; height: 12px; }
+    &:hover { border-color: #dc6263; }
+    &:hover &-x:after { @include close-icon(#dc6263); }
+  }
+  &-title { margin: 0; line-height: 1.42857143; font-family: OpenSans; font-size: 24px; font-weight: 600; }
+  &-body { padding: 0px 60px; }
```
Comment (apress-hound) [inline src/Dialog/rc-dialog.scss]: `0px` should be written without units as `0`

File: src/Dialog/example.js (hunk 15 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Unclosed regular expression.<br>Unrecoverable syntax error. (61% scanned).

File: src/Dialog/example.js (hunk 16 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 17 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 19 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 20 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Missing semicolon.

File: src/Dialog/example.js (hunk 21 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Unexpected '}'.<br>Expected an identifier and instead saw '}'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Expected an identifier and instead saw ';'.

File: src/Dialog/example.js (hunk 22 above)
Comment (apress-hound) [inline src/Dialog/example.js]: 'onClick' is defined but never used.

File: src/Dialog/example.js (hunk 26 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Expected ')' to match '}' from line 12 and instead saw 'onClose'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).<br>Expected an identifier and instead saw '='.<br>Expected an operator and instead saw '('.<br>Expected '(' and instead saw ')'.<br>Expected an identifier and instead saw '=>'.

File: src/Dialog/example.js (hunk 27 above)
Comment (apress-hound) [inline src/Dialog/example.js]: Expected '(' and instead saw '}'.

**Pull request merged**